assets/app/view/logo.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# rubocop:disable Layout/LineLength
+
+require 'lib/color'
+
+module View
+  class Logo < Snabberb::Component
+    needs :user, default: nil, store: true
+
+    def render
+      props = {
+        style: {
+          position: 'absolute',
+          top: '10px',
+          width: '80px',
+          height: '32px',
+        },
+        attrs: {
+          width: '500px',
+          height: '198px',
+          viewBox: '0 0 500 198',
+        },
+      }
+
+      bg_color = @user&.dig(:settings, :bg_color) || '#ffffff'
+      logo_id = bg_color == '#ffffff' ? 'logo--dark' : 'logo'
+      # green = '#71bf44'
+      offboard = '#ec232a'
+      yellow = '#F0E68C'
+      if Lib::Color.brightness(bg_color) < 125
+        polygon_fill = offboard
+        path_fill = '#eee'
+      else
+        polygon_fill = yellow
+        path_fill = '#000'
+      end
+
+      h('svg#logo', props, [
+        h(:title, '18xx.Games'),
+        h(:g, { attrs: { id: logo_id } }, [
+          h(:polygon, attrs: { id: 'logo__polygon', fill: polygon_fill, points: '250 0 335.736515 49.5 335.736515 148.5 250 198 164.263485 148.5 164.263485 49.5' }),
```

Comment:
it seems like there are only 2 colors? i thought there would be infinite many which is why we choose to make it a class, if it's just two colors, should we just make it 2 static svgs?

Reply:
The polygon has 2 colors only as of now, I could switch this to infinite if you’d wish.

Reply:
if it's only going to be 2 colors, i'd rather have it as 2 svgs. if you're going to do all this work to make it inline, might as well make use of it.

Reply:
Sure. Polygon still as backdrop or can I move it to the left of 18xx.Games?